reactDashboard/src/components/DataCharts/DataChartsUtils.test.js

```diff
@@ -11,8 +13,71 @@ import {
   LAST_THREE_MONTHS,
 } from "../DataCharts/DataChartsConsts";
 
+const TEST_MAX_DATA_1 = [
```

Comment:
Minor point - prettier is also happy with the following format - easier to read:
const TEST_MAX_DATA_1 = [
  { t: Date.parse("2020-01-01"), y: 363 },
  { t: Date.parse("2020-01-02"), y: 101 },
  { t: Date.parse("2020-01-03"), y: 257 },
  { t: Date.parse("2020-01-04"), y: 771 },
  { t: Date.parse("2020-01-05"), y: 799 },
  { t: Date.parse("2020-01-06"), y: 297 },
  { t: Date.parse("2020-01-07"), y: 118 },
];
const TEST_MAX_DATA_2 = [
  { t: Date.parse("2020-01-01"), y: 4056 },
  { t: Date.parse("2020-01-02"), y: 256 },
  { t: Date.parse("2020-01-03"), y: 235 },
  { t: Date.parse("2020-01-04"), y: 367 },
  { t: Date.parse("2020-01-05"), y: 2478 },
  { t: Date.parse("2020-01-06"), y: 163 },
  { t: Date.parse("2020-01-07"), y: 842 },
];